src/init.cpp

```diff
@@ -1356,15 +1355,27 @@ bool AppInitMain(const util::Ref& context, NodeContext& node, interfaces::BlockA
     // is not yet setup and may end up being set up twice if we
     // need to reindex later.
 
+    // see Step 2: parameter interactions for more information about these
```

Comment:
I think only `g_relay_txes` needs to be moved up here.

Reply:
Yeah but it made sense to keep them all grouped imho